apps/finance/app/src/components/DateRange/DateRangeInput.js

```diff
@@ -29,13 +29,20 @@ class DateRangeInput extends React.PureComponent {
   get formattedStartDate() {
     const { startDate } = this.state
 
-    return isDate(startDate) ? formatDate(startDate, this.props.format) : ''
+    return isDate(startDate)
+      ? formatDate(startDate, this.props.format)
+      : '--/--/----'
   }
 
   get formattedEndDate() {
-    const { endDate } = this.state
-
-    return isDate(endDate) ? formatDate(endDate, this.props.format) : ''
+    const { endDate, startDate } = this.state
+    const { format } = this.props
+
+    return isDate(endDate)
+      ? formatDate(endDate, format)
+      : startDate
+      ? '--/--/----'
```

Comment:
I wonder if we could do something so that screen readers don’t hear “dash-dash-slash-dash-dash…”? :thinking: Probably not a great solution, but maybe using `placeholder` with an empty value could help some readers to know that the field is empty, making it easy to skip the placeholder?

Reply:
I added this dashes to keep the input like this:
![image](https://user-images.githubusercontent.com/3072458/56560420-acd20e00-65a4-11e9-8818-fb29f88af001.png)

Instead of:
![image](https://user-images.githubusercontent.com/3072458/56560444-b9566680-65a4-11e9-9f3b-6b1a51cd2274.png)

Added the empty placeholder as suggested.
The thing is that if only one date is selected then the dashes fill in to keep char width. How do screen readers behave with empty spaces? (Do they read "space-space-space...")

Reply:
Ah yes, I forgot it was the same field for both… empty spaces shouldn’t be read no, do you see a solution using spaces?

Otherwise I think we can keep it how it was for now (everything in `value`), and, when we’ll move it to aragonUI, we’ll use an element on top of the input to ensure that it’s only seen as a visual thing.

Reply:
`const DATE_PLACEHOLDER = '          '`

![image](https://user-images.githubusercontent.com/3072458/56575477-16164900-65c6-11e9-91eb-0bbd2d5dea2a.png)

Reply:
I do think the dashes are more helpful and agree we can figure out a solution when implementing in `aragonUi`

Reply:
@bpierre the last commit (https://github.com/aragon/aragon-apps/pull/819/commits/96b23408f2cd62dd9d26c5402e39bd506d8bed7b) uses placeholder instead of value when either value is missing and value when it has both. Let me know if this ok and if so, can I get your approve to merge?

Reply:
Sorry I forgot to approve, done!